lib/connextclient/types.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * The configurable options for the connext client.
+ */
+
+export type ConnextClientConfig = {
+  disable: boolean;
+  ethProviderUrl: string;
+  nodeUrl: string;
+};
+
+/** General information about the state of this connext client. */
+export type ConnextInfo = {
+  status: string;
+  error?: string;
+  address?: string;
+  channels?: ConnextChannelCount;
+  chain?: string;
+  version?: string;
+};
+
+export type ConnextChannelCount = {
+  active: number;
+  settled: number;
+  closed: number;
+};
+
+/**
+ * The payload for the openChannel call.
+ */
+export type OpenChannelPayload = {
+  partner_address: string;
+  token_address: string;
+  total_deposit: number;
+  settle_timeout: number;
+};
+
+/**
+ * The connext version.
+ */
+
+export type ConnextVersion = {
+  version: string;
+};
+
+/**
+ * A connext payment channel.
+ */
+export type Channel = OpenChannelPayload & {
+  channel_address: string;
+  token_network_identifier: string;
+  channel_identifier: number;
+  /** The balance of the channel denominated in the smallest units supported by the token. */
+  balance: number;
+  state: string;
+};
+
+/**
+ * The response for tokenPayment call.
+ */
+export type TokenPaymentResponse = TokenPaymentRequest & {
+  secret?: string;
+};
+
+/**
+ * The payload for tokenPayment call.
+ */
+export type TokenPaymentRequest = {
```

Comment:
This type and its members (with the `snake_case` variable names) are specific to Raiden and there's no reason they need to be reused for Connext. I'd say it's best that Connext types and properties/names align with the names used natively by the Connext client and/or standard javascript/typescript naming conventions (which usually doesn't use snake_case).

Edit: Same thing applies to other types in this class.